homeworks/olha.tymchyshyn_olyatmchshn/4-basics/basics.js

```diff
@@ -0,0 +1,24 @@
+const a = 24 + 68;
+const b = 37 - 16;
+const c = 100 / 5;
+const x = 4 * 16;
+const y = 4 ** 2;
+const z = 23 % 5;
+
+const add = a + b;
+console.log(`Addition: ${a} + ${b} = ${add}`);
+
+const diff = b - c;
+console.log(`Subtraction: ${b} - ${c} = ${diff}`);
+
+const multiply = y * z;
+console.log(`Multiplication: ${y} * ${z} = ${multiply}`);
+
+const div = c / z;
+console.log(`Division: ${c} / ${z} = ${+div.toFixed(2)}`);
```

Comment:
```suggestion
console.log(`Division: ${c} / ${z} = ${div.toFixed(2)}`);
```
Hey, @olyatmchshn, there is no necessity to make a number out of `div.toFixed(2)` by using `+` in this case.

Reply:
Anyway it became a sting as a part of `Division: ${c} / ${z} = ${div.toFixed(2)}`